src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Processors/OperationFilteringTelemetryProcessor.cs

```diff
@@ -18,6 +21,24 @@ public OperationFilteringTelemetryProcessor(ITelemetryProcessor next)
 
         public void Process(ITelemetry item)
         {
+            // There are scenarios in Functions where we can have two TelemetryClients running for a short period.
+            // We use this property to ensure we only send one of the RequestTelemetry objects (auto-tracked by App 
+            // Insights) to the server.
+            if (item is RequestTelemetry request)
+            {
+                lock (_lock)
+                {
+                    if (request.Properties.ContainsKey(ProcessedKey))
+                    {
+                        return;
+                    }
+                    else
+                    {
+                        request.Properties[ProcessedKey] = "1";
```

Comment:
What is inspecting this property?

Reply:
This... directly above. if it's there, we stop processing it.

Reply:
I see the property being set if we don't return, but what actually "processes" the event/item?

Reply:
Nevermind... my issue with the diff